app/src/main/java/org/sopt/soptseminar_week1/utils/EmptyChecker.kt

```diff
@@ -0,0 +1,12 @@
+package org.sopt.soptseminar_week1.utils
+
+import android.text.Editable
+
+fun isAllEditTextFilled (list: List<Editable>) : Boolean{
+    for(item in list) {
+        if(item.isBlank()) {
+            return false
+        }
+    }
```

Comment:
- Collection이면 all 함수를 써도 될 것 같아
```suggestion
    if(list.all { it.isBlank() })
        return true
    return false
```

Reply:
그리고 저 함수는 모든 View에 대해서 다 동일한 기능을 제공할까? 만약 어떤 뷰에서 다 안채워져도 검증 통과시킨다고 한다면? 그런경우에는 이런 검증이 무력화되지 않을까? 검증의 로직은 각 뷰에서 구현해도 충분하다고 생각해

Reply:
안 채워져도 되는 `EditText.text`들은 매개변수로 안 넘기면 되지만 함수 이름을 all이라고 써둬서 모호하긴 하겠구만..
좋은 피드백 고맙다!!!!!!!👩‍💻